test_unstructured_ingest/test-ingest-biomed.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "$SCRIPT_DIR"/.. || exit 1
+
+if [[ "$(find test_unstructured_ingest/expected-structured-output/biomed-ingest-output/ -type f -size +10k | wc -l)" != 2 ]]; then
+    echo "The test fixtures in test_unstructured_ingest/expected-structured-output/biomed-ingest-output/ look suspicious. At least one of the files is too small."
+    echo "Did you overwrite test fixtures with bad outputs?"
+    exit 1
+fi
+
+PYTHONPATH=. ./unstructured/ingest/main.py    --biomed-api-from "2019-01-02" --biomed-api-until "2019-01-02+00:03:10" --structured-output-dir biomed-ingest-output  --num-processes 2 --verbose --download-dir biomed-download --preserve-downloads
+
+if ! diff -ru biomed-ingest-output test_unstructured_ingest/expected-structured-output/biomed-ingest-output ; then
+   echo
+   echo "There are differences from the previously checked-in structured outputs."
+   echo 
+   echo "If these differences are acceptable, copy the outputs from"
+   echo "biomed-ingest-output/ to test_unstructured_ingest/expected-structured-output/biomed-ingest-output/ after running"
+   echo 
+   echo "PYTHONPATH=. ./unstructured/ingest/main.py    --biomed-api-from "2019-01-02" --biomed-api-until "2019-01-02+00:03:10" --structured-output-dir biomed-ingest-output  --num-processes 2 --verbose --download-dir biomed-download --preserve-downloads"
+   echo
+   exit 1
+fi
```

Comment:
Please `chmod ugo+x` this script

Reply:
Hi @cragwolfe I'm not quite sure what you mean here. 

If I'm to chmod, my current understanding is that it has to be on the `test-ingest-biomed.sh` file just before it is called. At what point do I get to do that in our workflow here? Do I chmod in the `test-ingest.sh` file? I looked for other examples but can't seem to find other cases where such is being done in the codebase.

Reply:
Perhaps he means to run
```
git update-index --chmod=+x test_unstructured_ingest/test-ingest-biomed.sh
```
Which updates the files permissions to 755 on git, making it executable for everyone.

Reply:
Ah makes sense. I didn't even know that was possible. Thank you.